recipes/zarp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "zarp" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0defdec6fe8dabf66d40f2f4d16dfc2619f509f11763356908af3be51ca049ab
+
+build:
+  noarch: python
+  entry_points:
+   - zarp = zarp.cli:main
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - addict >=2.4.0
+    - bidict >=0.22.0
+    - email-validator >=1.2.1
+    - jsonref >=0.2
+    - pandas >=1.3.5
+    - pydantic >=1.9.2 
+    - pydantic-collections >=0.3.0
+    - pygments >=2.8.0
+    - python
+    - rich >=12.5.1
+    - pip
+    - setuptools-git >=1.2
+    - pyyaml
+  run:
+    - addict >=2.4.0
+    - bidict >=0.22.0
+    - email-validator >=1.2.1
+    - jsonref >=0.2
+    - pandas >=1.3.5
+    - pydantic >=1.9.2
+    - pydantic-collections >=0.3.0
+    - pygments >=2.8.0
+    - python
+    - rich >=12.5.1
+    - pyyaml
```

Comment:
Here i added `pyyaml`. In the original requirements we had `snakemake` as a dependency that installs 
`pyyaml` by default. Should I also add `snakemake` here or `pyyaml` is enough?